### PR TITLE
[DOCS] Fleet managed integrations content

### PR DESCRIPTION
**Pull request by amyjtechwriter**
Some Fleet installed integrations are now read-only and badged with a 'Managed' tag in the Kibana UI. 

system-managed
These are Elastic managed and cannot be edited. To be customized they must be copied first, which de-couples them from the original configuration. 

Relates to:[#175150](https://github.com/elastic/kibana/issues/175150) 
Corresponds with Kibana page update: [#178380](https://github.com/elastic/kibana/pull/178380)

### CODE REVIEW

**Comment by amyjtechwriter** [comment]
@elasticmachine run docs-build

File: docs/en/ingest-management/integrations/managed-integrations-content.asciidoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+++++
+<titleabbrev>Managed integrations content</titleabbrev>
+++++
+
+Most integration content installed by {fleet} isn’t editable. This content is tagged with a **Managed** badge in the {kib} UI. Managed content itself cannot be edited or deleted, however managed visualizations, dashboards, and saved searches can be cloned.   
+
+[role="screenshot"]
+image::images/system-managed.png[An image of the new managed badge.]
+
+The cloned content is unlinked with the original reference configurations, so becomes independent from the original managed content. For example, with a cloned dashboard, the cloned panels become entirely self-contained copy without any dependencies on the original configuration. Clones can be customized and modified without accidentally affecting the original.
```
Comment (drewdaemon) [inline docs/en/ingest-management/integrations/managed-integrations-content.asciidoc:13]: > The cloned content is unlinked with the original reference configurations, so becomes independent from the original managed content.

I don't think we can call this a general rule. For example, a managed visualization may use the `logs-*` data view which is a managed data view. And the cloned version will, too.

The only case where the unlinking happens is with dashboard panels, as you've described here!

Comment (amyjtechwriter) [reply docs/en/ingest-management/integrations/managed-integrations-content.asciidoc:13]: Ah ok, sorry that was a bad assumption. Let me rework this and also update the corresponding Kibana page.

File: docs/en/ingest-management/integrations/integrations.asciidoc [hunk 2]
```diff
@@ -61,3 +61,4 @@ The *Integrations* app in {kib} needs access to the public {package-registry} to
 discover integrations. If your deployment has network restrictions, you can
 {fleet-guide}/air-gapped.html#air-gapped-diy-epr[deploy your own self-managed {package-registry}].
 ====
+
```
Comment (kilfoyle) [inline docs/en/ingest-management/integrations/integrations.asciidoc:64]: I'd maybe remove this extra line just to avoid including an extra file in the PR.

**Review by kilfoyle — DISMISSED** [review]
Looks good to me so far! 🚀

**Review by kilfoyle — DISMISSED** [review]

File: docs/en/ingest-management/integrations/managed-integrations-content.asciidoc [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+image::images/system-managed.png[An image of the new managed badge.]
+
+With dashboards the cloned content is unlinked with the original reference configurations, so becomes independent from the original managed content. For example, with a cloned dashboard, the cloned panels become entirely self-contained copy without any dependencies on the original configuration. Clones can be customized and modified without accidentally affecting the original.
+
+[NOTE]
+====
+The cloned managed content retains the managed badge, but is independent from the original. 
+====
+
+With managed content relating to specific visualization editor such as Lens, TSVB, and Maps, the clones retain the original reference configurations. To clone the visualization, view it in the editor then becoming to make edits. Once finished editing you are prompted to save the edits as a new visualization. The same applies to editing any saved searches in a managed visualization.
```
Comment (drewdaemon) [inline docs/en/ingest-management/integrations/managed-integrations-content.asciidoc:20]: Typo?

> view it in the editor then becoming to make edits

File: docs/en/ingest-management/integrations/managed-integrations-content.asciidoc [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+++++
+<titleabbrev>Managed integrations content</titleabbrev>
+++++
+
+Most integration content installed by {fleet} isn’t editable. This content is tagged with a **Managed** badge in the {kib} UI. Managed content itself cannot be edited or deleted, however managed visualizations, dashboards, and saved searches can be cloned.   
+
+[role="screenshot"]
+image::images/system-managed.png[An image of the new managed badge.]
+
+With dashboards the cloned content is unlinked with the original reference configurations, so becomes independent from the original managed content. For example, with a cloned dashboard, the cloned panels become entirely self-contained copy without any dependencies on the original configuration. Clones can be customized and modified without accidentally affecting the original.
```
Comment (drewdaemon) [inline docs/en/ingest-management/integrations/managed-integrations-content.asciidoc:13]: Is this language about "reference configurations" standard in our docs? To me it seems more straightforward to say something like

"When a managed dashboard is cloned, any (library|linked|reference|something else) panels are made inline..."

This is just my opinion. I like your clarifications around what this actually means.

Also, is this a typo?

> the cloned panels become entirely self-contained copy

File: docs/en/ingest-management/integrations/managed-integrations-content.asciidoc (hunk 3 above)
Comment (amyjtechwriter) [reply docs/en/ingest-management/integrations/managed-integrations-content.asciidoc]: Oh yes it was supposed to say begin not becoming. Corrected is latest commit.

File: docs/en/ingest-management/integrations/managed-integrations-content.asciidoc (hunk 4 above)
Comment (amyjtechwriter) [reply docs/en/ingest-management/integrations/managed-integrations-content.asciidoc]: I've re-worded the start of the paragraph to:

File: docs/en/ingest-management/integrations/managed-integrations-content.asciidoc [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+++++
+<titleabbrev>Managed integrations content</titleabbrev>
+++++
+
+Most integration content installed by {fleet} isn’t editable. This content is tagged with a **Managed** badge in the {kib} UI. Managed content itself cannot be edited or deleted, however managed visualizations, dashboards, and saved searches can be cloned.   
+
+[role="screenshot"]
+image::images/system-managed.png[An image of the new managed badge.]
+
+When a managed dashboard is cloned, any linked or referenced panels become part of the clone without relying on external sources. The panels are integrated into the cloned dashboard as stand alone components. For example, with a cloned dashboard, the cloned panels become entirely self-contained copies without any dependencies on the original configuration. Clones can be customized and modified without accidentally affecting the original.
```
Comment (drewdaemon) [inline docs/en/ingest-management/integrations/managed-integrations-content.asciidoc:13]: Nice!

**Review by drewdaemon — APPROVED** [review]
LGTM

**Review by kilfoyle — APPROVED** [review]
LGTM! 🚀

**Pull request merged**